### DISCUSSION TITLE
auto-devices model setting in the gui vs at load-time

### DISCUSSION THREAD

**Discussion by quarterturn** [oobabooga/text-generation-webui#1769] [General]
I notice it's trouble-free running llama-30b-4bit-128g on my hardware (dual P100 16GB, 48 Xeon cores supporting AVX (no AVX2), and 128 GB RAM) with 'auto-devices' checked vs trying to specify it at launch on the command line, where it quickly hits a CUDA OOM exception.
Any ideas why that is?

**Comment by m-pokrovskii** [comment]
how did you run it without avx2? My CPU doesn't support AVX2 either.
